services/web/server/src/simcore_service_webserver/projects/_wallets_api.py

```diff
@@ -11,3 +14,25 @@ async def get_project_wallet(app, project_id: ProjectID):
     wallet_db: WalletDB | None = await db.get_project_wallet(project_uuid=project_id)
     wallet: WalletGet | None = WalletGet(**wallet_db.dict()) if wallet_db else None
     return wallet
+
+
+async def connect_wallet_to_project(
```

Comment:
MINOR: I try to reduce the verbs used in naming functions to the bare minimum in order to avoid ending up with a large collections of unprecise synonyms. For instance,  these are coming from the standard library
- `get`<-> `set`
- `add`<->`remove` 
- `load`<->`dump` 

IMO. Whe should priorize using those verbs over alternative synonyms (e.g. `inject`, `associate`, `acquire`, `connect`.. ) 

Here you are creating **a relation** between two resources, namely `project` and  `wallet`. You are associating one or more wallets to the project. If you allow more than one, I would use `add_` while if you only allow one then I would use `set_`. that is 
```
async def set_project_wallet(...)
   ...
   
async def add_project_wallet(...)
   ...

```

I do not mean that `connect` is "wrong" but it is an unnecessary synonym when the "known" verbs can make the job.

Reply:
Yes, you are right I agree with you. The thing is that in exactly the same fashion we are already connecting wallets to projects therefore to be consistent I would do it in the same matter (unless we change it everywhere)